libbeat/tests/compose/wrapper.go

```diff
@@ -371,6 +372,43 @@ func (d *wrapperDriver) serviceNames(ctx context.Context) ([]string, error) {
 	return strings.Fields(stdout.String()), nil
 }
 
+// Inspect a container.
+func (d *wrapperDriver) Inspect(ctx context.Context, serviceName string) (string, error) {
+	list, err := d.client.ContainerList(ctx, types.ContainerListOptions{All: true})
+	if err != nil {
+		return "", errors.Wrap(err, "listing containers to be inspected")
+	}
+
+	var found bool
+	var c types.Container
+	for _, container := range list {
+		aServiceName, ok := container.Labels[labelComposeService]
+		if ok && serviceName == aServiceName {
+			c = container
+			found = true
+			break
+		}
+	}
+
+	if !found {
+		return "", errors.Wrap(err, "container not found")
```

Comment:
No error to wrap here, `err` will be probably nil here.
```suggestion
		return "", errors.Errorf("container not found for service %s", serviceName)
```

Reply:
Fixed.